internal/handlers/pg/pgdb/indexes_test.go

```diff
@@ -39,11 +39,11 @@ func TestCreateIndexIfNotExists(t *testing.T) {
 	collectionName := testutil.CollectionName(t)
 	setupDatabase(ctx, t, pool, databaseName)
 
-	indexName := "test"
+	indexName := "test_nested_field"
 	err := pool.InTransaction(ctx, func(tx pgx.Tx) error {
 		idx := Index{
 			Name: indexName,
-			Key:  []IndexKeyPair{{Field: "foo", Order: types.Ascending}, {Field: "bar", Order: types.Descending}},
+			Key:  []IndexKeyPair{{Field: "foo.bar", Order: types.Ascending}},
```

Comment:
I like the test, and as now we have two tests with similar logic, it seems that it's time to introduce table tests here! ;)
So, let's create a map of test cases. It seems that we only need a couple of fields - index and expected definition. And then we can add a test cases without nested fields, a test case with one nested field, and maybe a test case with two nested fields (like `v.foo.bar`) just in case.